R/mergelist.R

```diff
@@ -0,0 +1,9 @@
+cbindlist = function(l, copy=TRUE) {
+  ans = .Call(Ccbindlist, l, copy)
+  if (anyDuplicated(names(ans))) { ## invalidate key and index
+    setattr(ans, "sorted", NULL)
+    setattr(ans, "index", integer())
```

Comment:
```suggestion
    setattr(ans, "index", NULL)
```
Since we also do not set `"index"` to `integer()` initially